tools/bazel-wrapper/flake.nix

```
@@ -36,8 +37,17 @@
             ++ lib.optionals stdenv.isDarwin [
               darwin.cctools
             ];
+          rc_line =
+            if "${system}" == "x86_64-linux" then
+              ''
+                common --shell_executable ${pkgs.lib.getExe pkgs.bash}
+              ''
+            else
+              "";
```

```suggestion
              ''
                common --shell_executable ${pkgs.lib.getExe pkgs.bash}
              '';
```

As far as I understand, this would also be necessary for aarch64-linux (although not currently included in systems).

I don't think it hurts to generate it for all systems, including darwin, but that's up to you.

---

Yeah, I agree it *shouldn't* harm anything... but if it does, I don't have a good way to debug. I removed the conditional logic.